apps/examples/lcd_test/example_lcd.c

```diff
@@ -360,6 +366,48 @@ static void test_fps(void)
 	}
 }
 
+#ifdef CONFIG_INPUT_IST415
```

Comment:
The config `CONFIG_INPUT_IST415` is a some hw specific config. It's not a common config for touch. But the code is common. The conditional should be a common config like config_input_touch.